tensor_test.go

```diff
@@ -123,3 +125,21 @@ func TestTensorPinMemory(t *testing.T) {
 		assert.Equal(t, " 1  2\n 3  4\n[ CPUFloatType{2,2} ]", b.String())
 	}
 }
+
+func TestTensorGC(t *testing.T) {
+	torch.GC()
+	runtime.LockOSThread()
+	c := make(chan bool, 0)
```

Comment:
To simulate `ImageLoader`  workload, maybe we shoul use the Tensor channel as the following code: 

``` golang
c := make(chan Tensor)
go func() {
    a := torch.NewTensor([][]float32{{1, 2}, {3, 4}})
    c <- a
    time.Sleep(time.Second)
    runtime.KeepAlive(&a)
}()
a := <- c
```

Reply:
Done.